test/functional/page_objects/visualize_page.js

```diff
@@ -206,7 +206,8 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
     }
 
     toggleSpyPanel() {
-      return testSubjects.click('spyToggleButton');
+      testSubjects.click('spyToggleButton');
+      return PageObjects.common.sleep(1000);
```

Comment:
We try really hard to avoid fixing tests with sleeps.  Perhaps instead we can change it so that instead of toggle it's either open or close, and wrap it in a retry.try and ensure that after the click, the panel is opened or closed, and if not, throw an error so the retry.try keeps trying.  I think that's the right way to fix this.